src/content/docs/en/guides/cms/kontent-ai.mdx

```diff
@@ -0,0 +1,514 @@
+---
+title: Kontent.ai & Astro
+description: Add content to your Astro project using Kontent.ai as a CMS
+type: cms
+service: Kontent.ai
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+
+[Kontent.ai](https://www.kontent.ai/) is the headless CMS that allows you to manage content in a structured and modular way, comes with AI capabilities, and delivers content to any platform you choose.
+
+## Integrating with Astro
+
+In this section, you'll use the [Kontent.ai TypeScript SDK](https://github.com/kontent-ai/delivery-sdk-js) to connect your Kontent.ai project to Astro.
+
+### Prerequisites
+
+To get started, you'll need the following:
+
+1. **Kontent.ai project** - If you don't have a Kontent.ai account yet, [sign up for free](https://app.kontent.ai/sign-up) and create a new project.
+
+2. **Delivery API keys** - You will need the Environment ID for published content and the Preview API key for fetching drafts (optional). Find them both in **Environment Settings -> API keys** tab in Kontent.ai.
+
+### Setting up credentials
+
+To add your Kontent.ai project credentials to Astro, create a `.env` file in the root of your project with the following variables:
+
+```ini title=".env"
+KONTENT_ENVIRONMENT_ID=YOUR_ENVIRONMENT_ID
+KONTENT_PREVIEW_API_KEY=YOUR_PREVIEW_API_KEY
+```
+
+Now, you can use these environment variables in your Astro project.
+
+### Installing dependencies
+
+To connect Astro with your Kontent.ai project, install the [Kontent.ai TypeScript SDK](https://github.com/kontent-ai/delivery-sdk-js):
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+    npm install @kontent-ai/delivery-sdk
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+    pnpm install @kontent-ai/delivery-sdk
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+    yarn add @kontent-ai/delivery-sdk
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Next, create a new file called `kontent.ts` in the `src/lib/` directory of your Astro project.
+
+```ts title="src/lib/kontent.ts"
+import { createDeliveryClient } from "@kontent-ai/delivery-sdk";
+
+export const deliveryClient = createDeliveryClient({
+    environmentId: import.meta.env.KONTENT_ENVIRONMENT_ID,
+    previewApiKey: import.meta.env.KONTENT_PREVIEW_API_KEY,
+});
```

Comment:
Maybe add a link here somewhere to https://docs.astro.build/en/guides/environment-variables/#getting-environment-variables, since people might use other runtimes and get lost on getting the variable..

Something like: `using another runtime? Read more on [getting environment variables in Astro](https://docs.astro.build/en/guides/environment-variables/#getting-environment-variables)`

Reply:
Added